src/Features/Core/Portable/SolutionCrawler/SolutionCrawlerLogger.cs

```diff
@@ -236,9 +238,9 @@ public static void LogIncrementalAnalyzerProcessorStatistics(int correlationId,
 
                     m[CreateProperty(propertyName, Max)] = result.Maximum;
                     m[CreateProperty(propertyName, Min)] = result.Minimum;
-                    m[CreateProperty(propertyName, Median)] = result.Median.Value;
+                    m[CreateProperty(propertyName, Median)] = result.Median!.Value;
```

Comment:
What's going on here?

Reply:
StatisticResult may or may not have Median calculated. In this case it does.